src/client/tile.cpp

```diff
@@ -537,6 +537,34 @@ static void apply_colorize(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 static void apply_multiplication(video::IImage *dst, v2u32 dst_pos, v2u32 size,
 		const video::SColor &color);
 
+// Perform a Screen blend with the given color. The opposite effect of a
+// Multiply blend.
+static void apply_screen(video::IImage *dst, v2u32 dst_pos, v2u32 size,
+		const video::SColor &color);
+
+// Adjust the hue, saturation, and lightness of destination. Like
+// "Hue-Saturation" in GIMP, but with 0 being the mid-point.
+// If colorize is true then all pixels will be converted to the specified hue,
+// but retain their saturation and lightness, like "Colorize" in GIMP.
+// Hue should be from -180 to +180, or from 0 to 360.
+// Saturation and lightness are both from -100 to +100
+static void apply_hue_saturation(video::IImage *dst, v2u32 dst_pos, v2u32 size,
+		int hue, int saturation, int lightness, bool colorize);
```

Comment:
Not that there's a real reason for this but graphics code usually uses `s32` for numbers